students/axel_nowaczyk/lesson_02_flow_control/The_number_of_elements_equal_to_the_maximum.py

```diff
@@ -0,0 +1,20 @@
+def number_of_elements_equal_to_the_maximum():
+    inputs = []
+    while True:
```

Comment:
Try to use other condition (without True and break)

Reply:
Similar condition I have found on official wiki.python.org website as common practice. Link:  https://wiki.python.org/moin/WhileLoop